Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as builder
+WORKDIR /go/src/github.com/gojekfarm/stevedore
+COPY ./ ./
+RUN make compile
+
+FROM alpine:latest
+ENV HELM_VERSION="v3.4.0"
+RUN wget -q https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz -O - | tar -xzO linux-amd64/helm > /usr/local/bin/helm
+RUN chmod +x /usr/local/bin/helm
+RUN helm repo add stable https://charts.helm.sh/stable
+RUN helm repo update
+
+COPY --from=builder /go/src/github.com/gojekfarm/stevedore/out/stevedore /usr/local/bin/
+WORKDIR /workdir
```

Comment:
Shall we use a less privileged user instead of root?

Reply:
yes, lets use less privileged user and`scratch` images over `alpine` image if possible

Reply:
Agree... Added changes to PR